src/PKSim.Core/Services/CKDDiseaseStateImplementation.cs

```diff
@@ -215,6 +215,11 @@ public void Validate(OriginData originData)
          return (false, PKSimConstants.Error.CKDOnlyAvailableForAdult);
       }
 
+      public void ApplyTo(IndividualMolecule individualMolecule)
+      {
```

Comment:
CKD does not do anything FOR NOW. There is a publication coming out soon influencing some enzymes and transporters concentrations